### PR TITLE
Chore: Add jQuery Foundation copyright

### PR DESCRIPTION
**Pull request by nzakas**

### CODE REVIEW

File: lib/ast-node-types.js [hunk 1]
```diff
@@ -1,28 +1,8 @@
...
- * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
- * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
- * ARE DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> BE LIABLE FOR ANY
- * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
- * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
- * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
- * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
- * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
- * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ * @copyright jQuery Foundation and other contributors, https://jquery.org/
```
Comment (ilyavolodin) [inline lib/ast-node-types.js]: Ariya's copyright removed because Esprima is part of the Foundation, right?

Comment (nzakas) [inline lib/ast-node-types.js]: Yup. It's the same copyright holder now.

**Comment by ilyavolodin** [comment]
LGTM

**Comment by nzakas** [comment]
I'm just going to hold off on merging so I can use this as a test case for the jQuery CLA check.

**Pull request merged**